agent/action/shop_buy_fav_controller.py

```
@@ -25,33 +36,45 @@
+# 点击间延迟（秒），等待菱形光特效消散
+CLICK_DELAY = 1.5
+# 验证前等待（秒），等待 Toast 消息淡出
+VERIFY_DELAY = 2.0
+# 单页对齐重试次数
+MAX_RETRIES = 1
+# 星-名几何配对窗口（星星右边缘 → 商品名左边缘）
 BIND_DX_MIN = 5    # 商品名至少在星星右侧 5px
 BIND_DX_MAX = 40   # 最远不超过 40px
 BIND_DY_MAX = 15   # Y 轴差距不超过 15px
 
+# ← ReadNames_Csm.attach（3/4·名识别参数）
```

Ruff (0.16.0)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)